tests/sqllogictests/suites/tpch/queries.test

```diff
@@ -1429,23 +2917,23 @@ HashJoin: INNER
 ├── Build
 │   └── HashJoin: INNER
 │       ├── Build
-│       │   └── Scan: default.tpch_test.nation (#4) (read rows: 25)
+│       │   └── HashJoin: INNER
+│       │       ├── Build
+│       │       │   └── Scan: default.tpch_test.nation (#5) (read rows: 50)
+│       │       └── Probe
+│       │           └── Scan: default.tpch_test.customer (#3) (read rows: 30000)
 │       └── Probe
-│           └── Scan: default.tpch_test.supplier (#0) (read rows: 1000)
+│           └── Scan: default.tpch_test.orders (#2) (read rows: 300000)
```

Comment:
It would be better to split the spilling test into another test because it will not affect the TPC-H join order.

Reply:
done